packages/runtime/src/helpers/config.ts

```diff
@@ -157,6 +160,14 @@ export const configureHandlerFunctions = async ({
       }
     })
   })
+
+  for (const apiLambda of apiLambdas) {
+    const { functionName, includedFiles } = apiLambda
+    // TODO: add all the nextRoot/wasm/excludedModules stuff from above
```

Comment:
Is this outstanding for this PR?

Reply:
it is, and i've got some tests to fix - will turn into a draft for now. Do you think the general direction makes sense, though?